tests/integ/test_packaging.py

```diff
@@ -501,7 +513,7 @@ def test_add_requirements_with_ranged_requirements_in_yaml(session, ranged_yaml_
     IS_IN_STORED_PROC,
     reason="Subprocess calls are not allowed within stored procedures",
 )
-def test_add_packages_unsupported_during_registration(session):
+def test_add_packages_unsupported_during_udf_registration(session):
     """
```

Comment:
Change test title to disambiguate between udf registration test and sproc registration test.